modules/javafx.graphics/src/shims/java/com/sun/javafx/css/BitSetShim.java

```diff
@@ -185,4 +185,8 @@ public Stream<T> stream() {
     public Stream<T> parallelStream() {
         return delegate.parallelStream();
     }
+
+    public boolean equals(Object other) {
```

Comment:
This line generates 2 warnings in Eclipse:

```
Description	Resource	Type	Path	Location
The method equals(Object) of type BitSetShim<T> should be tagged with @Override since it actually overrides a superclass method	BitSetShim.java	Java Problem	/graphics/src/shims/java/com/sun/javafx/css	line 189
The type BitSetShim<T> should also implement hashCode() since it overrides Object.equals()	BitSetShim.java	Java Problem	/graphics/src/shims/java/com/sun/javafx/css	line 39
```